packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -66,47 +68,30 @@ As `FlushMode.TurnBased` accumulates ops, it is the most vulnerable to run into
 
 Compression is relevant for both `FlushMode.TurnBased` and `FlushMode.Immediate` as it only targets the contents of the ops and not the number of ops in a batch. Compression is opaque to the server and implementations of the Fluid protocol do not need to alter their behavior to support this client feature.
 
-## Grouped batching
-
-The `IContainerRuntimeOptions.enableGroupedBatching` option has been added to the container runtime layer and is **off by default**. This option will group all batch messages under a new "grouped" message to be sent to the service. Upon receiving this new "grouped" message, the batch messages will be extracted and given the sequence number of the parent "grouped" message.
-
-The purpose for enabling grouped batching on top of compression is that regular compression won't include the empty messages in the chunks. Thus, if we have batches with many messages (i.e. more than 4k), we will go over the batch size limit just on empty op envelopes alone.
-
-See [below](#how-grouped-batching-works) for an example.
-
-### Risks
+Compressing a batch yields a batch of the same size, but only the first message has any content - the compressed payload containing all the original batch's content.
+The rest of the batch's messages are empty placeholders to reserve sequence numbers for the compressed messages.
```

Comment:
"Compressing a batch yields a batch of the same size" sounds a bit weird. Do you mean "before compression is applied"? I'd rephrase it and maybe say - "Compressing a batch first shifts all content into the first op, compressing it after that.

That said, I think we should consider simplifying even further (code & readme) and get rid of the mode where we send ops with empty content. Basically, both compression & op grouping are ON in main, and OFF in LTS. So, we can group them together into one modality and stop mentioning (and dealing with) a modality where compression is on, but op grouping is off.

Reply:
Yeah I like the idea of removing that modality. I've been mentally ignoring that case already.

AB#8124